tensorflow/python/ops/image_ops_impl.py

```diff
@@ -464,6 +464,14 @@ def flip_left_right(image):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+    
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
+    >> image = tf.random.normal(shape=(256, 256, 3))
+    >> tf.image.flip_left_right(image)
```

Comment:
```import tensorflow as tf``` statement isn't required.
Also doctests use three `>` signs i.e `>>>`; not two.